src/java/org/apache/cassandra/db/guardrails/Guardrail.java

```diff
@@ -75,6 +85,9 @@ protected void warn(String message)
 
     protected void warn(String message, String redactedMessage)
     {
+        if (skipNotifying(true))
```

Comment:
I'm a bit on the fence about omitting the client warning when `minNotifyIntervalInMs` is set. We want that property to not overflow server logs nor emit millions of diagnostic events when there are replicas with stuffed disks. But it might be legit to still send the warning to the client, although on the other hand we could also overflow client's logs if we emit client warnings for every write done when a node is stuffed.